monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -141,13 +143,12 @@ def export_to_disk(self, output_path: str, algo_name: str, **kwargs: Any) -> Non
             self.fill_records = self.fill_template_config(self.data_stats_files, self.output_path, **kwargs)
         logger.info(self.output_path)
 
-    def _create_cmd(self, train_params=None):
+    def _create_cmd(self, train_params: dict={}):
```

Comment:
Better not use an empty dict as the default because of [this](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python)

How about adding one line:
```
train_params = {} if train_params is None else train_params
```

Reply:
Thank you. Will change it

Reply:
> Hi @heyufan1995 , thanks for the explanation. Your proposal also makes sense. One possible concern I had was that the memory needs to hold (n_algos x n_files) before the ensemble takes place, which was the reason that I went one file by another. We may need to be careful and do something when there are lots of files to infer.
> 
> By the way, `infer_instance.predict` supports multiple files in one batch in the `predict_files` argument.

Thanks. I think we  just keep the current for loop.

Reply:
Thanks for pointing out. Will update